tests/test_api_admin.py

```diff
@@ -530,6 +530,38 @@ def test_get_disk_usage(self):
         self.assertIn('info', data)
         self.assertEqual(data['info'], 'Success')
 
+    @mock.patch('redisConnections.redisConnection.RedisConnection.get_module')
+    def test_get_redis_module_existing_module(self, mock_get_module):
+        mock_get_module.return_value = '{"name": "test_module"}'
+        response = self.client.get('/api/admin/module/test_module@2023-10-20/test_organization')
+        self.assertEqual(response.status_code, 200)
+        mock_get_module.assert_called_once_with('test_module@2023-10-20/test_organization')
```

Comment:
@pprovazn I would expect to at least check whether received JSON contains required `name`, `revision` and `organization`.
```python
response_data = response.get_json()
self.assertIn('name', response_data)
self.assertEquat(response_data['name], `test_module`)
...
```
Check whether you can update this on other test methods as well.